kuma/dashboards/urls.py

```diff
@@ -19,7 +22,7 @@
         views.topic_lookup,
         name='dashboards.topic_lookup'),
     url(r'^localization$',
-        shared_cache_control(RedirectView.as_view(
+        shared_cache_control(s_maxage=WEEK)(RedirectView.as_view(
```

Comment:
This could do with a code comment to explain. The truth is that it's exclusively an optimization thing that only works because of CloudFront.